packages/node-opcua-nodeid/test/test_nodeid.js

```diff
@@ -163,21 +163,21 @@ describe("testing coerceNodeId", function() {
         buffer.writeUInt32BE(0xb1dedada, 0);
         buffer.writeUInt32BE(0xb0b0abba, 4);
         const nodeId = coerceNodeId(buffer);
-        nodeId.toString().should.eql("ns=0;b=b1dedadab0b0abba");
-        nodeId.value.toString("hex").should.eql("b1dedadab0b0abba");
```

Comment:
Please keep :  nodeId.value.toString("hex").should.eql("b1dedadab0b0abba");

Reply:
reverted